src/project/qml/MuseScore/Project/internal/ScoreItem.qml

```diff
@@ -176,7 +177,7 @@ FocusScope {
                 text: root.name
 
                 wrapMode: Text.WrapAnywhere
-                maximumLineCount: 1
+                maximumLineCount: 2
```

Comment:
Does this not mean that the ScoreItem will become too tall and will therefore overlap with the item below it when there are so many recent scores that they appear in multiple rows?

Reply:
No?

![filetooltip-overlapping](https://user-images.githubusercontent.com/79419198/230784129-f55da514-9267-474a-9959-65129aab9ff5.png)